libjolie/src/jolie/lang/parse/SemanticVerifier.java

```diff
@@ -386,6 +387,9 @@ public void validate()
 		}
 	}
 
+	@Override
+	public void visit( ReThrowStatement n ) {}
```

Comment:
@fmontesi 
You are right! I have, without any luck, been trying to think of some cases where it actually would make sense to have a procedure that would be used in- and outside of a fault handler. However, the example would still be semantically correct as it wouldn't rethrow when called outside of the fault handler.

Another solution would be to print a warning at compile time whenever a procedure is referenced both inside and outside a fault handler, and then throw a RuntimeException if rethrow is executed outside.